### DISCUSSION TITLE
YoloV9 using torch

### DISCUSSION THREAD

**Discussion by AhmedAredah** [ultralytics/ultralytics#12636] [Docs]
I am trying to use yolo v9 by torch. The following code downloads v5:
model = torch.hub.load('ultralytics/yolov5', 'yolov5s', trust_repo=True)  # Loading a small version of YOLOv5

however, when I try to download v8 or v9, the files do not exist. Do they have different names? on ultralytics?

Thanks

**Comment by glenn-jocher** [comment]
Hello! It looks like you're trying to load YOLOv8 or YOLOv9 using a similar approach as YOLOv5. The models do use different repositories and methods for loading.

For YOLOv8, you can use:
```python
from ultralytics import YOLO
model = YOLO('yolov8n.pt')  # Load YOLOv8 model
```

For YOLOv9, which is experimental and might be hosted differently, you'd typically find the instructions directly on the specific project page or repository if available.

Make sure you check the [Ultralytics documentation](https://docs.ultralytics.com) or repositories for the most accurate and updated information.

Hope this helps! 😊
